addons/mail/static/src/models/activity/activity.js

```diff
@@ -256,6 +268,7 @@ function factory(dependencies) {
         }),
         category: attr(),
         creator: many2one('mail.user'),
+        creatorPartner: many2one('mail.partner'),
```

Comment:
Feels like one should be related to the other.

But it is strange even in python to have `request_partner_id` when we already have `create_uid` that does apparently the same thing.

To avoid confusion for now, I would rename the new field here to `request_partner_id` just like in python, and add dostring above it to explain what it does (can be inspired by the commit message introducing it).